eng/Packages.props

```diff
@@ -18,7 +18,7 @@
     <PackageReference Update="Microsoft.VisualStudio.Setup.Configuration.Interop" Version="1.16.30" />
     <PackageReference Update="Microsoft.Win32.Registry" Version="4.3.0" />
     <PackageReference Update="NuGet.Build.Tasks" Version="$(NuGetBuildTasksVersion)" />
-    <PackageReference Update="NuGet.Frameworks" Version="$(NuGetBuildTasksVersion)" />
+    <PackageReference Update="NuGet.Frameworks" Version="5.7.0-rtm.6710" />
```

Comment:
What does rtm stand for? And will this always be available?

Reply:
I think it means release to manufacturing?... (@wli3 is that right?) I don't see why it wouldn't be avaliable.

Reply:
ya, "release to manufacturing". In our case it is mostly mean this version is ready for try. And it should be stable.
although remember to remove line once nuget proper insertion is in.

Reply:
What do you mean proper nuget insertion? We're using reflection here so we won't need insertions into MSBuild.

Reply:
ah, i forgot that. you are right

Reply:
What's the fallback if it can't find 5.7.0-rtm-6710?

Reply:
I don't know that there is a fallback, but this dependency is [only for testing](https://github.com/microsoft/msbuild/pull/5234/files#diff-ea54c5eb95cb47249db3854a7632151eR22). Otherwise we use reflection and try to load the dll from its VS install location or next to the MSBuild dll. This logic is [here](https://github.com/microsoft/msbuild/pull/5234/files#diff-5af5289df4605c1a0a27f11cb43a8d23R27) in the last intrinsic functions PR I made.